tf/do/main.tf

```diff
@@ -97,8 +96,8 @@ resource "digitalocean_droplet" "harmony_node" {
     }
 
     provisioner "file" {
-        source      = "files/rclone.sh"
-        destination = "/root/do-user/rclone.sh"
+        source      = "files/rclone.conf"
+        destination = "/root/do-user/rclone.conf"
```

Comment:
Will rclone work if rclone.conf is not in the ~/.config directory?

Reply:
we have these two lines already in the current main.tf

`            
"mv -f rclone.conf /root/.config/rclone",
"mv -f rclone.sh ../",
`